src/components/App/App.js

```diff
@@ -16,10 +16,7 @@ import { getConfig } from "app/selectors";
 function App() {
   const { baseAppURL } = useSelector(getConfig);
   const disableAnalytics = localStorage.getItem("disableAnalytics");
-  if (
-    process.env.NODE_ENV === "production" &&
-    (disableAnalytics === undefined || disableAnalytics === "false")
-  ) {
+  if (disableAnalytics === undefined || disableAnalytics === "false") {
```

Comment:
Turns out there is another issue here with this...

```suggestion
  if (!disableAnalytics || disableAnalytics === "false") {
```